docs/app/app.md

```diff
@@ -0,0 +1,77 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: App
+--->
+
+# App
+
+**Status**: [Development][DocumentStatus]
+
+This document defines events and attributes related to client-side applications (e.g. web apps or mobile apps).
+
+## Attributes
+
+See the [app attributes](/docs/attributes-registry/app.md) registry for details.
+
+## Click Events
+
+### Event: `app.screen.click`
+
+<!-- semconv event.app.screen.click -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+**Status:** ![Development](https://img.shields.io/badge/-development-blue)
+
+The event name MUST be `app.screen.click`.
+
+This event represents an instantaneous click on the screen of an application.
+
+The `app.screen.click` event can be used to indicate that a user has clicked or tapped on the screen portion of an application. Clicks outside of an application's active area SHOULD NOT generate this event. This event does not differentiate between touch/mouse down and touch/mouse up. Implementations SHOULD give preference to generating this event at the time the click is complete, typically on touch release or mouse up. The location of the click event MUST be provided in absolute screen pixels.
+
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`app.screen.coordinate.x`](/docs/attributes-registry/app.md) | int | The x (horizontal) coordinate of a screen coordinate, in screen pixels. | `0`; `131` | `Required` | ![Development](https://img.shields.io/badge/-development-blue) |
+| [`app.screen.coordinate.y`](/docs/attributes-registry/app.md) | int | The y (vertical) component of a screen coordinate, in screen pixels. | `12`; `99` | `Required` | ![Development](https://img.shields.io/badge/-development-blue) |
+
+<!-- markdownlint-restore -->
```

Comment:
are these two attributes sufficient to conclude anything useful? Or would we expect this event to be reported in a scope of a span? 

I'd imagine you'd want to record `user`, `enduser` and/or application name, device/brower info, os type, etc, would these be recorded as resources? If so, would be great to mention the list of resources that would make it complete in a free-form markdown text.

Reply:
Yes, they tell you where the user clicked on a screen. That's useful. 

I think `user` or `enduser` or whatever could be added later as needed. The app, device, os, etc. are on the resource. Is there prior art that lists out a resource like you're asking? Does it have to be in this PR?

Reply:
let's postpone it until https://github.com/open-telemetry/semantic-conventions/pull/2123 lands - it adds a formal way to document the association